Source/WebKit/WebProcess/WebProcess.messages.in

```diff
@@ -97,7 +97,7 @@ messages -> WebProcess LegacyReceiver NotRefCounted {
     DestroyAutomationSessionProxy()
 
     PrepareToSuspend(bool isSuspensionImminent, MonotonicTime estimatedSuspendTime) -> ()
-    ProcessDidResume()
+    ProcessDidResume(Seconds hiddenPageThrottlingAutoIncreaseLimit)
```

Comment:
Can't we leverage the existing SetHiddenPageDOMTimerThrottlingIncreaseLimit() instead of reusing `ProcessDidResume()`? I know this is one less IPC call upon resume but I'm not a fan of processDidResume being involved in timer throttling details.